src/Auth/DigestAuthenticate.php

```diff
@@ -81,16 +81,14 @@ class DigestAuthenticate extends BasicAuthenticate
      */
     public function __construct(ComponentRegistry $registry, array $config = [])
     {
-        $this->_registry = $registry;
-
         $this->config([
             'realm' => null,
             'qop' => 'auth',
             'nonce' => uniqid(''),
             'opaque' => null,
         ]);
 
-        $this->config($config);
+        parent::__construct($registry, $config);
```

Comment:
Seeing this change I remember I have already fixed this and few other things on the `3.next` branch. It would be best if this PR was targeted to `3.next` instead of `master`.